packages/ra-ui-materialui/src/list/ListView.tsx

```diff
@@ -70,7 +66,11 @@ export const ListView = <RecordType extends RaRecord = any>(
                       })
                     : children}
             </Content>
-            {pagination !== false && pagination}
+            {error ? (
+                <PaginationLimit message="ra.notification.data_provider_error" />
```

Comment:
It feels weird to me to render a component named 'pagination' here, since we don't even know if the data is supposed to be paginated or not.

I understand that the code of the component you want to display is very similar to the code of `PaginationLimit`, but to me they have different purposes, so they should be two different components (even if the code is duplicated).

Reply:
How about adding an `Error` component:

![error](https://user-images.githubusercontent.com/8268610/198324372-e3ff72c2-fb44-4caa-83d2-daf43bf048ce.png)

Reply:
I'm not too found of this solution either, cause we don't do it for other dataProvider errors, so why only do it for the list?

To me, the content of the `PaginationLimit` was enough for our needs (provided we change the message of course), but I just thought we should create a separate component for this use.

However, this `Error` component makes me realize it could be a welcome addition to add a 'Retry' button, that would call `refresh` from `useRefresh`. This could be useful in case it's only a temporary error.

What do you think?

Reply:
I like the idea of the Error page. Displaying an empty page with a warning is less user friendly IMHO.

Reply:
alright then :sweat_smile: